lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8075,25 +8074,25 @@ void ClangImporter::Implementation::collectMembersToAdd(
 
   SwiftDeclConverter converter(*this, CurrentVersion);
 
-  SmallVector<ProtocolDecl *, 4> protos = takeImportedProtocols(D);
-  if (auto clangClass = dyn_cast<clang::ObjCInterfaceDecl>(objcContainer)) {
+  if (auto clangProto = dyn_cast<clang::ObjCProtocolDecl>(objcContainer))
```

Comment:
I won't ask you to revert this part, but in the future I'd prefer not performing either of these actions (reordering if conditions, de-bracing an if) in a refactoring patch. They just add churn (to review for NFC-ness, and potentially conflict in later merges) without improving subsequent readability.

Reply:
Yes, in my more cautious re-redo, I kept the ordering the same. Not sure if I de-braced, but you make a great point! I will be more gentle in the future.